### PR TITLE
Changing domain manager permissions

### PR DESCRIPTION
**Pull request by melinoix**
Domain manager permissions are changed to allow to create reference control inside his domain/sub-domains

### CODE REVIEW

File: backend/core/startup.py
Comment (nas-tabchiche) [inline backend/core/startup.py:1]: The task's subject was to allow the creation, edition and deletion of a `ReferenceControl`, not `AppliedControl`. Furthermore, `view_appliedcontrol` is present twice in the list. Once on [line 132](https://github.com/intuitem/ciso-assistant-community/blob/a379d48dd178d6af9203da86143a917a72f2bae0/backend/core/startup.py#L132) and once on line 179.
For the first point, after reading **my own** comment under the ticket, I realized I wrote `_appliedcontrol`. My mistake. Still, a domain manager has no business managing applied controls 😉

**Review by nas-tabchiche — CHANGES_REQUESTED** [review]
### To do

Change `{view,change,delete}_appliedcontrol` to `{view,change,delete}_referencecontrol`.

File: backend/core/startup.py
Comment (eric-intuitem) [reply backend/core/startup.py:1]: Let's keep it simple: a domain manager can do anything in its domain. The single exception is approval.
| Domain manager      | full access to selected domains (except approval), in particular managing rights for these domains. Read access to global objects |

**Comment by nas-tabchiche** [comment]
Closing this in favor of #893

**Pull request closed without merging**